tests/webserver_webserver-deployment_test.yaml

```diff
@@ -7,3 +7,25 @@ tests:
     asserts:
       - isKind:
           of: Deployment
+  - it: should get correct airflow container image for 1.10.11
+    set:
+      webserver:
+        useDefaultAirflowImage: false
+      airflowVersion: 1.10.11
+      images:
+        airflow:
+          tag: 1.10.11-custom
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[0].image
+          value: quay.io/astronomer/ap-airflow:1.10.11-custom
+  - it: should get correct airflow container image for 2.0.0
+    set:
+      webserver:
+        useDefaultAirflowImage: true
```

Comment:
Could there be a test case where you dont specify any value for useDefaultAirflowImage at all and it should assume the default of true, and hence there will be a restart? 

Not a blocker for merge. Just a curious question.

Reply:
default value is true https://github.com/astronomer/airflow-chart/pull/157/files#diff-8377b3e3740a3fcd9f682e5fb55425f2fdbece1791854b9e5013e7f1a5e60e7eR284